portal-kernel/src/com/liferay/portal/kernel/util/WebKeys.java

```diff
@@ -495,7 +495,7 @@ public interface WebKeys {
 		"PORTLET_RENDER_PARAMETERS_";
 
 	public static final String PORTLET_RESOURCE_STATIC_URLS =
-		"PORTLET_RESOURCE_STATIC_URLS";
+		"LIFERAY_SHARED_PORTLET_RESOURCE_STATIC_URLS";
```

Comment:
@dsanz, BChan will most likely ask for a change of variable, Or a new variable in any case...